### PR TITLE
Item enums

### PR DESCRIPTION
**Pull request by ShermanA-13**
__x__ Wrote Tests
__x__ Implemented
__x__ Reviewed

## Necessary checkmarks:
- [x] All Tests are Passing
- [x] The code will run locally

## Type of change
- [] New feature
- [x] Bug Fix

## Implements/Fixes:
* Add enum category and values
* Refactor tests to include enum values  

## Check the correct boxes
- [x] This broke nothing
- [x] All Tests are Passing
- [x] The code will run locally

## Testing Changes
- [] Tests have been added
- [] No Tests have been changed
- [x] Some Tests have been changed
- [x] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code
- [x] I have removed any testing syntax (save_and_open_page/pry) in my spec files

### CODE REVIEW

File: spec/requests/api/v1/items_request_spec.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@
       expect(item[:attributes][:name]).to be_a(String)
       expect(item[:attributes][:description]).to be_a(String)
       expect(item[:attributes][:count]).to be_an(Integer)
-      expect(item[:attributes][:category]).to be_an(Integer)
+      expect(item[:attributes][:category]).to be_an(String)
```
Comment (phillipkamps) [inline spec/requests/api/v1/items_request_spec.rb:20]: I thought we're dealing with integers on the BE for enums, why are we expecting a string here?

File: spec/requests/api/v1/items_request_spec.rb [hunk 2]
```diff
@@ -35,11 +35,11 @@
       name: "Organic Crash Pad",
       description: "Super soft and thicc, heavy though",
       count: 1,
-      category: 7,
+      category: "Crash Pads",
```
Comment (phillipkamps) [inline spec/requests/api/v1/items_request_spec.rb:38]: Same question here as above. I thought the BE database is storing ints not strings? Maybe I don't understand enums...

**Review by phillipkamps — APPROVED** [review]
Gotta have @ShermanA-13 break it down for me, but seems good!

**Comment by sueboyd922** [comment]
Looks good. finally kind of understanding the enums!

**Pull request merged**